geometric/molecule.py

```diff
@@ -2,7 +2,6 @@
 from __future__ import division
 from __future__ import print_function
 
-import copy
```

Comment:
`copy` is used several places in `molecule.py`.  Why is it being removed here?

Reply:
This is interesting. My IDE said it wasn't needed.

The import of `copy` on that line is being overwritten by a later import, where there is `from .PDB import *`. Since the `copy` module is imported there, the `*` also brings it to `molecule.py`. (Also why the tests still pass).

I can add it back though for safety

Reply:
Ah, thanks.  Let's add it back because `molecule.py` is designed to work as a standalone module.